### PR TITLE
Savery 2

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: journal/modernized/05-chapter-4.adoc [hunk 1]
```diff
@@ -280,12 +318,19 @@ Never shall your memory be effaced from my soul, I shall not cease to love you;
...
 
+[.signed-section-signature]
 Marconnay.
 
+[.signed-section-context-close]
 Berlin, 22nd October, 1796.]
 
+--
+
 Returning now to the Journal:
```
Comment (Henderjay) [inline journal/modernized/05-chapter-4.adoc:335]: I think this phrase "Returning to the Journal" would look best if we put the [.offset] tag above it. That will keep it from being indented, and then put a space below it before the journal picks back up.

File: journal/original/05-chapter-4.adoc [hunk 2]
```diff
@@ -280,12 +318,19 @@ Never shall your memory be effaced from my soul, I shall not cease to love you;
...
 
+[.signed-section-signature]
 Marconnay.
 
+[.signed-section-context-close]
 Berlin, 22nd October, 1796.]
 
+--
+
 Returning now to the Journal:
```
Comment (Henderjay) [inline journal/original/05-chapter-4.adoc:335]: here too... this would look best with an [.offset]

File: journal/modernized/08-chapter-7.adoc [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 Testimony Of the Monthly Meeting of Friends of Philadelphia for the Northern District,
 concerning William Savery.
+//Does this need a sub-heading or anything?
```
Comment (Henderjay) [inline journal/modernized/08-chapter-7.adoc]: yeah, lets use a [.blurb] for this. LIke this:

```ASCIIDOC
[.blurb]
=== Testimony of the Monthly blah blah
```

both here and in the original.

**Review by Henderjay — CHANGES_REQUESTED** [review]
Just a few things to change. thanks!

File: journal/original/05-chapter-4.adoc (hunk 2 above)
Comment (maijapapaya) [reply journal/original/05-chapter-4.adoc:335]: I totally agree. I've been looking back between these corrections of how they were and it's really nice to get a visual of the difference between them.

**Comment by Henderjay** [comment]
merged

**Pull request merged**